app/forms/sufia/forms/batch_edit_form.rb

```diff
@@ -1,9 +1,46 @@
 module Sufia
   module Forms
-    class BatchEditForm < CurationConcerns::Forms::FileSetEditForm
-      self.terms = [:resource_type, :title, :creator, :contributor, :description,
-                    :keyword, :rights, :publisher, :date_created, :subject, :language,
+    class BatchEditForm < CurationConcerns::Forms::WorkForm
+      self.terms = [:creator, :contributor, :description,
```

Comment:
I see that `resource_type`, `title`, and `date_created` are no longer included in the form's `terms`. Should those no longer be batch-editable?